components.json

```
@@ -238,6 +238,12 @@
 			"require": "c",
 			"owner": "zeitgeist87"
 		},
+		"cfscript": {
+			"title": "CFML Script",
```

The docs consistently call this langauge "CFScript". I think we should too.
```suggestion
			"title": "CFScript",
```